extension/popup/popupController.jsx

```diff
@@ -50,6 +50,16 @@ this.popupController = (function() {
     const init = async () => {
       const userSettings = await settings.getSettings();
       if (!userSettings.collectTranscriptsOptinAnswered) {
+        console.log("Searching for open onboarding tabs...");
+        for (const tab of await browser.tabs.query({
+          url: ["moz-extension://*/onboarding/onboard.html"]
```

Comment:
Note you can also use `browser.runtime.getURL("/onboarding/onboard.html")` to get the exact URL.